cli/jobs/train/r/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+- R
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to submit R jobs to an AzureMLCompute Cluster using the AzureML 2.0 CLI.
+---
+
+# R using the AzureML 2.0 CLI
+
+This directory provides some sample projects to run R using the AzureML 2.0 CLI.
+
+## Environment
+
+All the projects in this directory use the `rocker/tidyverse:4.0.0-ubuntu18.04` image from Docker Hub. This image has the tidyverse and its dependencies installed (see [Rocker](https://github.com/rocker-org/rocker) for more details).
```

Comment:
will this remain true over time (i.e. will we switch it to a curated image/environment we maintain?) - if so would remove